mirrord/analytics/src/lib.rs

```diff
@@ -66,6 +67,31 @@ impl Analytics {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AnalyticsHash(#[serde(with = "serde_base64")] Vec<u8>);
```

Comment:
I have no idea why this has to be base64, so maybe add docs explaining?

Reply:
`Vec<u8>` serializes really badly to json so base64